CMakeLists.txt

```diff
@@ -304,9 +303,9 @@ ExternalProject_Add(rocksdb
   -DWITH_ZSTD=ON
   -DWITH_GFLAGS=ON
 )
-set(ROCKSDB_LIBRARIE ${INSTALL_LIBDIR}/librocksdb.a)
-set(ROCKSDB_INCLUDE_DIR ${INSTALL_INCLUDEDIR})
-set(ROCKSDB_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/${EP_BASE_SUFFIX}/Source/rocksdb)
+# set(ROCKSDB_LIBRARIE ${INSTALL_LIBDIR}/librocksdb.a)
+# set(ROCKSDB_INCLUDE_DIR ${INSTALL_INCLUDEDIR})
+# set(ROCKSDB_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/${EP_BASE_SUFFIX}/Source/rocksdb)
```

Comment:
ROCKSDB_SOURCE_DIR is used for bw compile, so that it can find `file/file_util.h`

Reply:
ok, I will add it